PROJECT_GUIDELINES.md

```diff
@@ -0,0 +1,228 @@
+# Project Guidelines
+
+Hey there and thanks for stopping by! We welcome all feedback and contributions to [Calcite App Components](https://github.com/Esri/calcite-app-components/blob/master/README.md). If you are interested in getting involved, we ask that you give our guidelines a quick read through. Below is a list of things we've found work well for us as we continue developing our components.
+
+## TOC
+
+- [Code of conduct](https://github.com/Esri/contributing/blob/master/CODE_OF_CONDUCT.md)
+- [Contribution guidelines](https://github.com/esri/contributing)
+- [Conventions](#conventions)
+  - [Formatting](#formatting)
+  - [Github](#github)
+  - [Report a bug!](#report-a-bug)
+- [Dist/package](https://github.com/Esri/calcite-app-components/blob/master/GETTING_STARTED.md_)
+- [Code base](#code-base)
+  - [A11y](#a11y)
+  - [Components](#components)
+- [Gotchas](#gotchas)
+
+## Conventions
+
+### Formatting
+
+This project uses lint-staged to automatically format code on commit, making it easier to contribute.
+
+### Github
+
+#### Issues
+
+We have created [templates](https://github.com/Esri/calcite-app-components/issues/new/choose) for new issues for everyone to follow for consistency. Please be specific and thorough when submitting new issues! Think about things like user stories, acceptance criteria, design, and any other details that might be helpful for developers and designers.
+
+#### Commits
+
+This project follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/), which are used to generate the changelog. Be sure to provide clear and sufficient information in commit messages. This is important because the commit messages (type `feat` and `fix`) are used to automatically update the changelog. For ease of discoverability, commit messages for breaking changes should use both the header (`!`) and body (`BREAKING CHANGE:`) syntax:
```

Comment:
Suggestion: Drop `(type `feat` and `fix`)` to avoid treating these types as the only ones that should be clear and expressive.